### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PashaD9**
Update index.html

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,12 +6,183 @@
...
+      <fieldset>
+        <legend>Additional info</legend>
+        <label class="field">
+          Comments:
+          <textarea
+            name="text"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+         ></textarea>
```
Comment (tetiana-ivanchenko) [inline src/index.html]:
```suggestion
          <textarea
            name="text"
            cols="20"
            rows="2"
            autocomplete="off"
         >
         </textarea>
```

File: src/index.html [hunk 2]
```diff
@@ -6,12 +6,183 @@
...
+        </label>
+      </fieldset>
+
+      <button type="submit">Submit</button>
+
+    </form>
+    <script
+      type="text/javascript"
+      src="./main.js"
+   ></script>
```
Comment (tetiana-ivanchenko) [inline src/index.html]:
```suggestion
    <script
      type="text/javascript"
      src="./main.js"
   >
   </script>
```

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
GJ, just small fixes, also add your demo ant test-report links

**Comment by PashaD9** [comment]
https://pashad9.github.io/layout_html-form/
https://pashad9.github.io/layout_html-form/report/html_report/

**Pull request closed without merging**